torchaudio/transforms.py

```diff
@@ -1720,6 +1720,13 @@ def __init__(
         online: bool = False,
     ):
         super().__init__()
+        warnings.warn(
+            "The MVDR module is deprecated and will be removed in release 0.12. "
```

Comment:
Here are some thoughts around this deprecation.

1. We are constrained to give two release cycles for deprecation and removal for stable features. (MVDR was released as beta features.)
2. I am not sure if we can release the new replacements in the minor release `0.11.1`. And my preference is to postpone until `0.12`.
3. My understanding is that after introducing the new features and refactoring the internal, this MVDR module is in free form. It is not like the existence of this deprecated MVDR module will prevent the development some other features.

So I think we do not need to be eager about removal in 0.12. Once the replacements are introduced, we can leave it there, until we meet the two release cycle deprecation criteria, even though MVDR is not stable feature.

Reply:
After conforming with @Emrys365, we don't have to push the new features to minor release and can still adopt them to ESPNet. So It's fine to postpone them to `0.12`.
About the MVDR module, we can leave it before we add new modules in `transforms`. Then we can come back to this PR and deprecate it.